input/resources/parentCollection.StructureDefinition.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Aren't all the resources generated from Shorthand? If so, they should not be checked in.

Reply:
Some (in fact, most) resources are from the pre-FSH era and therefore checked in as JSON. I might try to convert them to FSH using GoFSH, but figured that should be an extr PR.

Reply:
Ok I see. But why are they all created newly? Didn't they exist before?